README.md

```diff
@@ -63,9 +63,27 @@ puzzle --mode gemini view -i path/to/gemini_database.db
 Puzzle uses the ped file to show more information in family view and in variant calls:
 
 ```
-puzzle view -i tests/fixtures/hapmap.vcf --family_file/-f tests/fixtures/hapmap.vcf
+puzzle view -i tests/fixtures/hapmap.vcf --family_file/-f tests/fixtures/hapmap.ped
 ```
 
+## Establish a persistent local database
+
+Puzzle can establish a persistent local database, so you can keep work between sessions.
+This enables other key features, such as gene lists for filtering and variant triage.
+First, you need to initialize a local database. Then one or more cases can be loaded and 
+are then ready for viewing as soon as the web server is started.
+
+```
+puzzle init --root tests/fixtures
```

Comment:
Thanks for your feedback Guille!

1. Not really. All I'm saying is to reuse existing sqlite databases, not creating new ones. No (hard) dependencies, just reuse.
2. That's a (bad) hypothetical fix on a current limitation that might be solved in the future. I think there's ways to just drop gemini tables and keep puzzle's.

You can still visualize VCF's in a nice way, all I'm proposing is that in order to persist a session, we do it in an existing database, not create a new, separate one.